.github/workflows/OSX.yml

```diff
@@ -1,7 +1,21 @@
 name: OSX
 on:
+  workflow_call:
+    inputs:
```

Comment:
Can this be duplicate eliminated and/or do we need both call and dispatch?

Reply:
welcome to gh actions 🥲 https://github.com/orgs/community/discussions/39357

Reply:
Tricked Mark into asking this question, since this works as documentation.

We can get rid of one of the two, possibly workflow_dispatch that can be implemented using workflow_call + helper workflow, BUT I think it's also good to keep this duplication [for now?].